### PR TITLE
Upgrade Podman to 1.2

### PR DESCRIPTION
**Pull request by RA489**
Upgrade Podman to 1.2.0
fixes https://github.com/kubernetes/minikube/issues/4181

### CODE REVIEW

File: deploy/iso/minikube-iso/package/podman/podman.mk [hunk 1]
```diff
@@ -1,4 +1,4 @@
-PODMAN_VERSION = v1.0.0
+PODMAN_VERSION = v1.2.0
```
Comment (tstromberg) [inline deploy/iso/minikube-iso/package/podman/podman.mk]: deploy/iso/minikube-iso/package/podman/podman.hash will also need to be updated.

You can verify that the build works by typing `make out/minikube.iso`: https://github.com/kubernetes/minikube/blob/master/docs/contributors/minikube_iso.md

Comment (afbjorklund) [reply deploy/iso/minikube-iso/package/podman/podman.mk]: You also need to update the git commit, it is used by the versioning system.

See https://github.com/containers/libpod/releases

Most likely, this upgrade needs to be timed with the system `runc` upgrade.

The plan is to use the _same_ binary for docker/podman/cri-o, hope it works...

**Comment by afbjorklund** [comment]
@RA489 : were you going to complete this upgrade, or did you want me to do it ?

Should also look into updating cri-o, preferrably to 1.14 - but otherwise `v1.13.9`

**Comment by RA489** [comment]
@afbjorklund i will complete this asap

**Comment by afbjorklund** [comment]
@RA489 : You can leave it, included it with the CRI-O upgrade in #4205

**Comment by RA489** [comment]
@afbjorklund closing in favour of  https://github.com/kubernetes/minikube/pull/4205

**Pull request closed without merging**